--- ntl-11.3.2/src/DoConfig~	2018-11-16 01:33:03.000000000 +0000
+++ ntl-11.3.2/src/DoConfig	2019-03-31 00:48:08.710718777 +0000
@@ -143,7 +143,7 @@
       }
    }
 
-   die "Error: unrecognized option: $arg\ntype \"./configure -h\" for help\n";
+#   die "Error: unrecognized option: $arg\ntype \"./configure -h\" for help\n";
 
 }
 
--- ntl-11.3.2/src/DoConfig~	2019-03-31 00:48:08.000000000 +0000
+++ ntl-11.3.2/src/DoConfig	2019-03-31 00:50:54.108716474 +0000
@@ -661,10 +661,6 @@
    CheckFlag('-pthread');
 }
    
-if ($native_flag) {
-   CheckFlag('-march=native');
-}
-
 # disable floating-point optimizations that are not value safe
 # for most compilers, the default at optimzation  '-O2' is to 
 # disable such optimizations. icc is the big exception,
--- ntl-11.3.2/src/DoConfig~	2019-03-31 00:50:54.000000000 +0000
+++ ntl-11.3.2/src/DoConfig	2019-03-31 00:51:29.673715979 +0000
@@ -27,10 +27,10 @@
 'LDLIBS'      => '-lm',
 'CPPFLAGS'    => '',
 
-'DEF_PREFIX'  => '/usr/local',
+'DEF_PREFIX'  => '/usr/',
 
 'PREFIX'      => '$(DEF_PREFIX)',
-'LIBDIR'      => '$(PREFIX)/lib',
+'LIBDIR'      => '$(PREFIX)/lib64',
 'INCLUDEDIR'  => '$(PREFIX)/include',
 'DOCDIR'      => '$(PREFIX)/share/doc',
 
--- ntl-11.3.2/src/DoConfig~	2019-03-31 00:51:29.000000000 +0000
+++ ntl-11.3.2/src/DoConfig	2019-03-31 01:00:55.967708095 +0000
@@ -14,7 +14,7 @@
 %MakeVal = (
 
 'CXX'	      => 'g++',
-'CXXFLAGS'    => '-g -O2',
+'CXXFLAGS'    => '-g -O2 $(CFLAGS)',
 'CXXAUTOFLAGS'=> '',
 'NOCONTRACT'  => '',
 'AR'	      => 'ar',
